src/praisonai/praisonai/cli/commands/gateway.py

```
@@ -113,6 +147,116 @@ def gateway_channels(
+                output.print_info("Starting the service...")
+                from praisonai.daemon import get_daemon_status
+                status = get_daemon_status()
+                if status.get("running"):
+                    output.print_success("Service is now running")
+                else:
+                    output.print_warn("Service installed but not running. Check system logs.")
+        else:
+            error = result.get("error", "Installation failed")
+            output.print_error(f"Installation failed: {error}")
+            raise typer.Exit(1)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)